src/message_data/test.rs

```diff
@@ -31,7 +31,7 @@ fn set() {
             .unwrap();
         assert!(!key_exists, "unhashed key should not exist in redis");
         let value: String = test.redis_client.get(test.internal_key.as_str()).unwrap();
-        assert_eq!(value, "wibble");
+        assert_eq!(value, "\"wibble\"");
```

Comment:
So will this change the serialization of existing datatypes, meaning an existing db will now contain invalid data?  Or were all uses of this already storing a JSON string in practice?

Reply:
Strictly speaking it's a breaking change. Data already stored would now fail to be read.

But actually there's no data in there at the moment because the auth server isn't posting any to `/send` yet. And even if there was, we could probably still get away with this because the reads are only in the queues process right now, which we still haven't enabled (so we could just drop any data if there was some).